integrations/salesforce/src/misc/utils/apiUtils.ts

```diff
@@ -6,7 +6,7 @@ export const makeRequest = async (
   input: Input,
   accessToken: string
 ): Promise<AxiosResponse> => {
-  return await axios({
+  return axios({
```

Comment:
Why was await removed?

Reply:
await is not really necessary if you are returning the promise in an async call, no issues were caused by this but I removed the redundance, do you want to keep it?

Reply:
No, I was just curious